### PR TITLE
[5.0] "action()" could also generate relative urls

### PR DESCRIPTION
**Pull request by appcia**

### CODE REVIEW

File: src/Illuminate/Foundation/helpers.php [hunk 1]
```diff
@@ -26,14 +26,14 @@ function abort($code, $message = '', array $headers = array())
 {
 	/**
 	 * Generate a URL to a controller action.
-	 *
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/helpers.php]: no

Comment (appcia) [inline src/Illuminate/Foundation/helpers.php]: what's the problem?

**Comment by GrahamCampbell** [comment]
Please squash to one commit.

**Comment by appcia** [comment]
done

**Pull request merged**